# SPDX-FileCopyrightText: Huawei Inc. 
# 
# SPDX-License-Identifier: Apache-2.0 
 
START_ABILITY_TIMEOUT is 3000ms which is too short in our case, causing some  
testcases in ActsBmsModuleUsageRecordTest to exceed timeout.  
Increasing the timeout to 30 seconds. 
 
Upstream-Status: Pending 
 
Signed-off-by: Francesco Pham <francesco.pham@huawei.com> 

diff --git a/appexecfwk/bundle_standard/bundlemanager/actsbmsmoduleusagerecordtest/entry/src/main/js/test/ActsBmsModuleUsageRecordTest.test.js b/appexecfwk/bundle_standard/bundlemanager/actsbmsmoduleusagerecordtest/entry/src/main/js/test/ActsBmsModuleUsageRecordTest.test.js
index d4068900a..1510b6608 100644
--- a/appexecfwk/bundle_standard/bundlemanager/actsbmsmoduleusagerecordtest/entry/src/main/js/test/ActsBmsModuleUsageRecordTest.test.js
+++ b/appexecfwk/bundle_standard/bundlemanager/actsbmsmoduleusagerecordtest/entry/src/main/js/test/ActsBmsModuleUsageRecordTest.test.js
@@ -24,7 +24,7 @@ const BUNDLE_NAME = 'com.example.third1';
 const NUM_TWO = 2;
 const NUM_TEN = 10;
 const INVALID_NUM = -1;
-const START_ABILITY_TIMEOUT = 3000;
+const START_ABILITY_TIMEOUT = 30000;
 const START_RECORD = 900;
 const TIMEOUT = 3000;
 
